packages/crackql/PKGBUILD

```diff
@@ -0,0 +1,46 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=crackql
+_pkgname=CrackQL
+pkgver=53.f4d41cd
+pkgrel=1
+pkgdesc='GraphQL password brute-force and fuzzing utility'
+arch=('any')
+groups=('blackarch' 'blackarch-webapp' 'blackarch-exploitation' 'blackarch-fuzzer')
+url='https://github.com/nicholasaleks/CrackQL'
+license=('BSD 3-Clause License')
```

Comment:
I think you should just tag BSD as per https://wiki.archlinux.org/title/PKGBUILD#license